ui/eslint.config.js

```diff
@@ -8,12 +8,7 @@ const config = defineConfig([
   ...createConfig(opts),
   {
     rules: {
-      'jsx-a11y/anchor-is-valid': 'off',
-      'jsx-a11y/control-has-associated-label': 'off',
-      'react/destructuring-assignment': 'off',
-      'react/no-multi-comp': 'off',
```

Comment:
Forbid having multiple components in the same file, so very minor. The whole file structure of the `ui/src` folder needs a bit of a review, so no point in creating more component files in arbitrary locations. Better to think about it on a case-by-case basis when refactoring.